ddtrace/__init__.py

```diff
@@ -1,4 +1,38 @@
+import sys
+
+
+def global_excepthook(tp, value, traceback):
+    """The global tracer except hook."""
+    if asbool(os.getenv("DD_INSTRUMENTATION_TELEMETRY_ENABLED", True)):
+        telemetry_writer.add_error(1, repr(value))
+        telemetry_writer.enable()
+
+
+_ORIGINAL_EXCEPTHOOK = sys.excepthook
+
+
+def _excepthook(tp, value, traceback):
+    global_excepthook(tp, value, traceback)
+    if _ORIGINAL_EXCEPTHOOK:  # type: ignore[comparison-overlap]
+        return _ORIGINAL_EXCEPTHOOK(tp, value, traceback)
+
+
+def install_excepthook():
+    """Install a hook that intercepts unhandled exception and send metrics about them."""
+    sys.excepthook = _excepthook
+
+
+def uninstall_excepthook():
+    """Uninstall the global tracer except hook."""
+    sys.excepthook = _ORIGINAL_EXCEPTHOOK
```

Comment:
nit: Can we define these helps in some utils/helper module. Would this lead to circular imports

Reply:
Before it was in the tracer module, which did lead to circular imports, not sure if utils/helper module would have the same result.